.circleci/config.yml

```diff
@@ -2,16 +2,9 @@
 
 version: 2
 references:
-  container_python36: &container_python36
+  container_python310: &container_python310
     docker:
-      - image: circleci/python:3.6.2
-      - name: localstack
-        image: localstack/localstack
-    working_dir: ~/project
-
-  container_python38: &container_python38
-    docker:
-      - image: circleci/python:3.8.1
+      - image: circleci/python:3.10
```

Comment:
While both exist, I _think_ we want `cimg/python:3.10`, at least that's what was done in cumulus-message-adapter and what I'm doing in cumulus-message-adapter-python.

Reply:
I think we still want to test both python 3.8 and 3.10 as in cumulus-message-adapter-python

Reply:
I'm folding 3.8 back in